### PR TITLE
GitHub Pages workflow updated

### PR DESCRIPTION
**Pull request by venkatajagannath**

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+aenum
+apache-airflow
+pydantic
```
Comment (pankajastro) [inline docs/requirements.txt:3]: do we need these dependencies for doc build?

File: .github/workflows/docs.yml [hunk 2]
```diff
@@ -1,29 +1,37 @@
 name: Deploy Sphinx documentation to Pages
 
-# Runs on pushes targeting the default branch
 on:
   push:
-    branches: [main]
-# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+    branches: [main, ghpages]
```
Comment (pankajastro) [inline .github/workflows/docs.yml]:
```suggestion
    branches: [main]
```
change before merging

**Review by pankajastro — DISMISSED** [review]
looks good. just posted some minor comments around dependencies

File: .github/workflows/docs.yml (hunk 2 above)
Comment (venkatajagannath) [reply .github/workflows/docs.yml]: changed

File: docs/requirements.txt (hunk 1 above)
Comment (venkatajagannath) [reply docs/requirements.txt]: removed

**Review by pankajastro — APPROVED** [review]

**Pull request merged**